README.adoc

```diff
@@ -167,6 +167,8 @@ You can access the application by making requests to the `gateway` job endpoints
 |Get a specific job |GET \http://localhost:8080/api/jobs/{jobId} |`{"jobId":"661891cb-ad36-4ef4-9bb3-641f973f2964","result":5}`
 |===
 
+Before any jobs have been created, the JSON output will be this `{"count":0,"results":[]}`. The `count` attribute is 0 because there are no jobs yet, and the `results` attribute is empty for the same reason.
```

Comment:
I think we want to try fix this issue programatically. Perhaps changing the `OptionalDouble.empty()` to `OptionalDouble.of(0)` in the `JobList.java` class, when there's no jobs being created. Can you try that?

Reply:
That does work, but it was discussed and in the end was decided to just include a paragraph instead

https://github.com/OpenLiberty/guide-microprofile-rest-client-async/issues/23

Reply:
The sentence itself needs to be edited.  Also, what we need to call is the difference between the 2 cases (return results)?

Fwiw, I am still on the fence on whether we should fix this.

Reply:
Ah ok, didn't see the discussion in the issue previously. IMO, displaying a 0 average is fine. It eliminates the confusion of having to explain the two different return results.

Reply:
I wouldn't put this as the first paragraph in the section either.  It will confuse users.  I will put it after the following 2 paragraphs as a clarification and before the concluding paragraph for the section.